### PR TITLE
Update README guide for Xcode 11 changes

### PR DESCRIPTION
**Pull request by yhkaplan**
This addresses: https://github.com/Carthage/Carthage/issues/2897

Here's what the pulldown menu looks like for your reference
スクリーンショット 2020-02-19 10 25 02

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Additionally, you'll need to copy debug symbols for debugging and crash reportin
 
 1. Create a [Cartfile][] that lists the frameworks you’d like to use in your project.
 1. Run `carthage update`. This will fetch dependencies into a [Carthage/Checkouts][] folder, then build each one or download a pre-compiled framework.
-1. On your application targets’ _General_ settings tab, in the “Linked Frameworks and Libraries” section, drag and drop each framework you want to use from the [Carthage/Build][] folder on disk.
+1. On your application targets’ _General_ settings tab, in the "Frameworks, Libraries, and Embedded Content" section, drag and drop each framework you want to use from the [Carthage/Build][] folder on disk. Then, in the "Embed"  section, select "Do Not Embed" from the pulldown menu for each item you added.
```
Comment (yhkaplan) [inline README.md]:
```suggestion
1. Open your application targets’ _General_ settings tab. For Xcode 11.0 and higher, in the "Frameworks, Libraries, and Embedded Content" section, drag and drop each framework you want to use from the [Carthage/Build][] folder on disk. Then, in the "Embed"  section, select "Do Not Embed" from the pulldown menu for each item added. For Xcode 10.x and lower, in the "Linked Frameworks and Libraries" section, drag and drop each framework you want to use from the [Carthage/Build][] folder on disk.
```

**Comment by yhkaplan** [comment]
Hey, can someone with permission merge this? Thanks in advance!

**Comment by yhkaplan** [comment]
@tmspzz Can you merge this? Thanks in advance!

**Comment by tmspzz** [comment]
LGTM 👍

**Pull request merged**